src/main/java/com/transportcompany/transportcompany/controllers/OrderController.java

```diff
@@ -0,0 +1,43 @@
+package com.transportcompany.transportcompany.controllers;
+
+import com.transportcompany.transportcompany.models.dtos.OrderDTO;
+import com.transportcompany.transportcompany.models.entities.Order;
+import com.transportcompany.transportcompany.services.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/orders")
+public class OrderController {
+
+    @Autowired
```

Comment:
If we do constructor injection, we don t need autowired on the property. You can use this annotation on the constructor.